contrib/utilities/run_clang_tidy.sh

```diff
@@ -41,9 +41,13 @@ if test ! -d "$SRC/source" -o ! -d "$SRC/include" -o ! -d "$SRC/examples" -o ! -
 fi
 echo "SRC-DIR=$SRC"
 
+# make sure that CC and CXX are unset
+unset CC
+unset CXX
+
 # enable MPI (to get MPI warnings)
 # export compile commands (so that run-clang-tidy.py works)
-ARGS=("-D" "DEAL_II_WITH_MPI=ON" "-D" "CMAKE_EXPORT_COMPILE_COMMANDS=ON" "-D" "CMAKE_BUILD_TYPE=Debug" "$@")
+ARGS=("-DMPI_INCLUDE_DIRS=/usr/include/x86_64-linux-gnu/mpich" "-D" "DEAL_II_WITH_MPI=ON" "-D" "CMAKE_BUILD_TYPE=Debug" "$@")
```

Comment:
@tjhei Somehow `FindMPI.cmake` is unable to extract an include path with clang. This works around the issue.